specs/schemas/tests/struct.yml

```diff
@@ -0,0 +1,96 @@
+schema: |
+  type SimpleStruct struct {
+    foo Int
+    bar Bool
+    baz String
+  }
+expected: |
+  {
+    "types": {
+      "SimpleStruct": {
+        "struct": {
+          "fields": {
+            "foo": {
+              "type": "Int"
+            },
+            "bar": {
+              "type": "Bool"
+            },
+            "baz": {
+              "type": "String"
+            }
+          },
+          "representation": {
+            "map": {}
+          }
+        }
+      }
+    }
+  }
+blocks:
+  - actual: |
+      {
+        "foo": 100,
+        "bar": true,
+        "baz": "this is baz yo"
+      }
+    expected: |
+      {
+        "foo": 100,
+        "bar": true,
+        "baz": "this is baz yo"
+      }
+  - actual: # is this OK? string-to-int
```

Comment:
Oh, gosh.  I know these are probably coming from the earlier files, but the answer is "no" -- I don't think this kind of coercion is desired behavior at all.